csv/readcsv.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	runtime "github.com/ahsanfayaz52/EmployeeManagementSystem"
+	"github.com/ahsanfayaz52/EmployeeManagementSystem/models"
+)
+
+func main() {
+	rt, err := runtime.NewRuntime()
+	if err != nil {
+		panic(err)
+	}
+	root, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	filepath := "/Files/records.csv"
+	path := root + filepath
+	var rows = ReadCsv(path)
+	AddDataToDatabase(rows, rt)
+}
+
+// AddDataToDatabase adds employee data from csv to database.
+func AddDataToDatabase(rows [][]string, rt *runtime.Runtime) string {
+	for _, row := range rows {
```

Comment:
```
	for _, row := range rows {
		_, err := rt.Service().AddEmployee(&models.Employee{
                   Age: strconv.Atoi(row[2]),
                   Name:  row[0],
                   Address: row[1],
                   Salary:  strconv.Atoi(list[3]),
                   Phone: strconv.Atoi(list[4]),
                   ....
                })
`

I am guess it can be as simple as that...

The code should be as minimal as possible.

Reply:
done